troubleshoot/index.md

```diff
@@ -57,14 +57,44 @@ If you have an [Elastic subscription](https://www.elastic.co/pricing), you can c
   If you contact us by email, use the email address you registered with so we can help you more quickly. If your registered email is a distribution list, you can register a second email address with us. Just open a case to let us know the name and email address you want to add.
   :::
 
+  :::{warning}
+  All cases opened by email are opened at normal severity. For incidents, kindly ensure to raise via the [Elastic Support Portal](https://support.elastic.co/) at the [appropriate severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+  :::
+
 ## Working with support [troubleshoot-work-with-support]
 
-Try these tips when opening a support case:
+When opening a support case, kindly be sure to:
+
+* Describe the problem along with its [contextual severity](https://www.elastic.co/support/welcome#what-to-say-in-a-case). 
+
+  * If an error message was encountered, include full error message and timezone-designated dates and times of when the problem occurred.
+
+  * If the problem is UI-related and does not replicate via API, pull a [browser network log](https://www.elastic.co/blog/generating-browser-har-file-kibana-troubleshooting). 
+
+* Upload the related product's diagnostics and debug logs
 
-* Include the deployment ID that you want help with, especially if you have several deployments.
+  * If {{serverless-full}} or {{ech}}, Support can pull diagnostics on your behalf as long as you include resource ids:
+    
+    * {{serverless-full}} Project ID or {{kib}} URL
+    * {{ech}} Deployment ID or {{kib}} URL
 
-  You can find the deployment ID on the overview page for your cluster in the {{ecloud}} Console.
+  * If hosting on {{ece}} or {{eck}}:
```

Comment:
Does "Support can pull diagnostics on your behalf as long as you include resource ids:" apply here too? It's not clear to me if it does or doesn't.

Reply:
The resource ids apply below that ECE you have to "flagging `--deployments` as applicable" and ECK does pull everything it can see by default.